doc/Makefile

```
@@ -7,29 +7,25 @@ SPHINXAUTOGEN = sphinx-autogen
 BUILDDIR      = _build
 
 # Internal variables.
-ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
+ALLSPHINXOPTS = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
 
-.PHONY: help clean html linkcheck doctest api
-
-all: html
+.PHONY: help api html server clean
```

@seisman add all here?
```suggestion
.PHONY: help all api html server clean
```

---

Yes, see https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html